{
  try {
    executableSection.execute(restTestExecutionContext);
  }
 catch (  Exception e) {
    throw new RuntimeException(errorMessage(executableSection,e),e);
  }
catch (  AssertionError e) {
    throw new AssertionError(errorMessage(executableSection,e),e);
  }
}
